packages/ketchup/src/f-components/f-typography/f-typography.tsx

```diff
@@ -0,0 +1,61 @@
+import { FunctionalComponent, h, VNode } from '@stencil/core';
+import { FImage } from '../f-image/f-image';
+import { FImageProps } from '../f-image/f-image-declarations';
+import { FTypographyProps, FTypographyType } from './f-typography-declarations';
+
+/*-------------------------------------------------*/
+/*                C o m p o n e n t                */
+/*-------------------------------------------------*/
+
+export const FTypography: FunctionalComponent<FTypographyProps> = (
+    props: FTypographyProps,
+    children: VNode[]
+) => {
+    if (!props.type) {
+        props.type = FTypographyType.HEADING1;
+    }
+    if (!props.value) {
+        return;
+    }
+    if (!props.toolbar) {
```

Comment:
props.toolbar si boolean; set to false is different than not set, but here will go into the if if undefined OR false